src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Configuration.php

```diff
@@ -165,6 +179,14 @@ public function tolerates(array $deprecationGroups)
         return true;
     }
 
+    /**
+     * @return bool
+     */
+    public function isIgnoredDeprecation(Deprecation $deprecation)
```

Comment:
You can assume PHP 7.1: https://github.com/symfony/symfony/blob/4834d9e224b97f946fe8e13b2ba403bbc212c4ad/src/Symfony/Bridge/PhpUnit/composer.json#L19-L21
```suggestion
    public function isIgnoredDeprecation(Deprecation $deprecation): bool
```